google_maps/lib/main.dart

```diff
@@ -38,10 +38,16 @@ class _MyAppState extends State<MyApp> {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      title: 'Flutter Demo',
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+        useMaterial3: true,
+      ),
       home: Scaffold(
         appBar: AppBar(
           title: const Text('Sydney'),
           backgroundColor: Colors.green[700],
+
```

Comment:
I'm curious what this white space is for. I suspect it'll fail format check.

Reply:
> 

Oh it's just by mistake. Let's me fix